sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -938,7 +939,9 @@ struct _pi_queue : _pi_object {
   // Map of all command lists used in this queue.
   pi_command_list_map_t CommandListMap;
   // TODO: Assign Graph related command lists to command_graph object
+#if SYCL_EXT_ONEAPI_GRAPH
   pi_command_list_map_t LazyCommandListMap;
+#endif
```

Comment:
I feel like we should probably avoid guarding members like  unless required, since it would break ABI. If it doesn't affect compilation I'd say it's fine to leave it present but unused.